apps/studio/components/ui/NoSearchResults.tsx

```diff
@@ -2,26 +2,29 @@ import { Button, cn } from 'ui'
 
 export interface NoSearchResultsProps {
   searchString: string
+  withinTableCell?: boolean
   onResetFilter?: () => void
   className?: string
 }
 
 export const NoSearchResults = ({
   searchString,
+  withinTableCell = false,
```

Comment:
Otherwise we are constantly a bunch of classes via `className` whenever this component is used inside. table. This way we can simply render those classes when it is “standalone”.

I think a clearer prop would be the inverse of how I’ve done it: `standalone={true}`. But this is a move towards that without needing to touch all 15 instances.